lib/publiccloud/gcp_client.pm

```diff
@@ -44,7 +44,7 @@ sub init {
         $self->vault(publiccloud::vault->new());
         $self->create_credentials_file();
     }
-    assert_script_run('source ~/.bashrc');
+    assert_script_run("export GOOGLE_APPLICATION_CREDENTIALS=" . CREDENTIALS_FILE);
```

Comment:
Not for this PR but this constant should be renamed to `$CREDENTIALS_FILE`, as it is otherwise considered a Bareword and this would fail if we would use `strict subs` (which we should).